### ISSUE TITLE
make doc-nits failure

### ISSUE THREAD

**Issue by kroeckx** [openssl/openssl#8277]
I'm getting:
```
$ make doc-nits
(cd .; /usr/bin/perl util/find-doc-nits -n -p ) >doc-nits
doc/man7/openssl_user_macros.pod:1: Bad =over 


doc/internal/man3/DEFINE_SPARSE_ARRAY_OF.pod:1: OPENSSL_SA missing from NAME section
doc/internal/man3/DEFINE_SPARSE_ARRAY_OF.pod:1: DEFINE_SPARSE_ARRAY_OF missing from SYNOPSIS
doc/internal/man3/openssl_ctx_get_data.pod:1: openssl_ctx_free_index missing from SYNOPSIS
doc/internal/man3/openssl_ctx_get_data.pod:1: openssl_ctx_set_data missing from SYNOPSIS
doc/internal/man3/OSSL_METHOD_STORE.pod:1: OSSL_METHOD_STORE (filename) missing from NAME section
doc/internal/man3/OSSL_METHOD_STORE.pod:1: OSSL_METHOD_STORE missing from NAME section
make: *** [Makefile:524: doc-nits] Error 1
```

**Comment by levitte** [comment]
#8270, and apparently #8280 :wink:

**Comment by levitte** [comment]
Fixed by #8270
